src/tests/ftest/deployment/agent_failure.yaml

```diff
@@ -0,0 +1,42 @@
+hosts:
+    test_servers:
+        - server-A
+    test_clients:
+        - client-A
```

Comment:
Can we please use the large hardware cluster available in CI to make sure it works well in multiple server/client environment.